Volume/Views/WeeklyDebriefView.swift

```diff
@@ -98,6 +101,12 @@ struct WeeklyDebriefView: View {
                     .foregroundColor(.volume.orange)
                     .font(.helveticaBold(size: 16))
             }
+            .padding(.horizontal, 20)
+            .padding(.vertical, 10)
+            .overlay(RoundedRectangle(cornerRadius: 26)
+                .stroke(Color.volume.orange, lineWidth: 2))
```

Comment:
@dishanest wudya think?
```suggestion
            .overlay(
                RoundedRectangle(cornerRadius: 26)
                    .stroke(.volume.orange, lineWidth: 2)
            )
```
Also, not sure if we can implicitly get from Color ¯\_(ツ)_/¯